.changeset/poor-clouds-fix.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hono/schema-validator': major
```

Comment:
I think using `scheme-` is not suitable for this middleware name since `schema` is too general a phrase. So, how about `effect-validator` or `effect-schema-validator`?

Reply:
Exactly,  I was having hard time trying to decide the name. 

Given that in the future there is space for also a separate Schema OpenAPI validator too (based on [schema-openapi](https://github.com/sukovanej/schema-openapi)),
and following current Zod middleware package naming, although its a bit long, the best name is `effect-validator` so I am gonna rename to it.

Reply:
Hey @gunta 

`effect-validator` sounds good. Can you rename it?